src/Classes/ServiceAPI/MyRadio_Timeslot.php

```diff
@@ -348,6 +348,17 @@ public static function getMostListened($date = 0)
         return $top;
     }
 
+    public static function getFromSlug($slug)
```

Comment:
yeah probably, and this is pretty dirty as I didn't even limit it to the correct metadata type